pkgs/build-support/flutter/default.nix

```diff
@@ -223,20 +227,25 @@ let
 
     # binaries need to be patched
     autoPatchelf -- "$depsFolder"
-
-    runHook postConfigure
   '';
 
-  buildPhase = ''
-    runHook preBuild
+  preConfigurePhases = [ "flutterConfigurePhase" ];
 
+  # avoid cmake phase
+  configurePhase = "true";
```

Comment:
```suggestion
  dontConfigure = true;
```

Reply:
The user might want to overwrite configurePhase and then it would be skipped. Also using preConfigurePhases not sure if this will work.

Reply:
What happens if we just leave configurePhase and don't do anything to it? Maybe we just want to disable cmake and ninja?